### PR TITLE
Add myself to OWNERS

### PR DESCRIPTION
**Pull request by kyessenov**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -19,4 +20,5 @@ approvers:
   - mandarjog
   - frankbu
   - linsun
-  - hklai
```
Comment (ldemailly) [inline OWNERS]: you really want to remove @hklai  ?

Comment (kyessenov) [reply OWNERS]: nope, bad merge.

**Pull request merged**